root/travis.yml

```diff
@@ -0,0 +1,12 @@
+language: ruby
+cache: bundler
+rvm:
+- 2.2.1
+before_script:
+- bundle exec rake db:create --all
+- bundle exec rake db:migrate
+script:
+- bundle exec rake ci:tests
+
+services:
+- postgresql
```

Comment:
Travis generator usually don't indent.. but it's better (at least I prefer it) indented..